services/web/src/jest-test/Jest.test.tsx

```diff
@@ -0,0 +1,25 @@
+const renderer = require('react-test-renderer');
```

Comment:
Please rework the test to use @testing-library/react. The current test uses react-test-renderer and manually triggers callbacks, which can be simplified with @testing-library/react.

Reply:
fixed in 961ed7d27f063f29b78bec23e10ef3c792dd7a15